plugins/scaffolder-backend/src/scaffolder/stages/templater/cra/index.ts

```diff
@@ -19,7 +19,8 @@ import { TemplaterBase, TemplaterRunOptions } from '..';
 import path from 'path';
 import { TemplaterRunResult } from '../types';
 import * as yaml from 'yaml';
-import { GITHUB_ACTIONS_ANNOTATION } from '@backstage/plugin-github-actions';
+// eslint-disable-next-line monorepo/no-internal-import
```

Comment:
Right, this was another blocker for this addition ._.

We might want to either try to switch out that rule, or contribute a fix

Reply:
Asked the maintainer if this is something he wants - https://github.com/azz/eslint-plugin-monorepo/issues/10